posts/2022/esds-event-prep.md

```diff
@@ -0,0 +1,54 @@
+---
+author: Julia Kent
+date: 2022-11-02
+tags: tutorial hackathon events
+---
+
+# Preparation for a (Re)Introduction to Earth System Data Science (ESDS) Across NCAR/UCAR/UCP
+
+ESDS is hosting a (Re)Introduction to Earth System Data Science (ESDS) Across
+NCAR/UCAR/UCP on **November 10th and 11th** (Thursday-Friday). This event will
+be hybrid, in-person in NCAR's Mesa Lab Main Seminar Room and virtual over Zoom
+or Google Meet. More information is in this [previous post](esds-fall-event.md).
+
+## Environment Install
+
+### Binder
+
+This tutorial is designed to be run from Binder. You can do that by navigating to the [GeoCat Tutorial Repository](https://github.com/NCAR/geocat-tutorials) and selecting ["Launch Binder"](https://mybinder.org/v2/gh/NCAR/geocat-tutorials/main) in the `README.md`.
+
+Essentially, Binder is a web service that hosts Jupyter Notebooks in an immediately executable and reproducible way. [Learn more about Binder](https://mybinder.org/). For more information on Jupyter Notebooks, review the [Jupyter Project Pythia content](https://foundations.projectpythia.org/foundations/getting-started-jupyter.html).
+
+### Local Install
+
+For those of you who prefer to work on your local machine, please install Miniconda and Jupyter before the ESDS event.
+
+You can follow the [Project Pythia instructions to install Python on Jupyter](https://foundations.projectpythia.org/foundations/jupyter.html).
+
+If you have trouble installing miniconda - try to uninstall and reinstall it, making sure you select the correct install location.
+
+Once installed, you will
```

Comment:
Would you like to replace the following instructions with a referral to https://github.com/NCAR/geocat-tutorials/blob/main/README.md as it already gives local config instructions?

Reply:
Done!